install_config/upgrading/manual_upgrades.adoc

```diff
@@ -1613,142 +1613,145 @@ common issues:
+the installation content by means of a system such as Red Hat Satellite. This
 can be accomplished by sharing the consumer ID from the previous host.
 
 On the old host:
 ----
-subscription-manager identity | grep system
+# subscription-manager identity | grep system
 system identity: 6699375b-06db-48c4-941e-689efd6ce3aa
 ----
 
 On the new host:
 ----
-subscription-manager register --consumerid=6699375b-06db-48c4-941e-689efd6ce3aa
+# subscription-manager register --consumerid=6699375b-06db-48c4-941e-689efd6ce3aa
 ----
 
 [IMPORTANT]
 ====
-When the deployment has been declared successful, it is important that the old
-host be unregistered with `*subscription-manager clean*` to prevent the
-environment from being out of compliance.
+When the deployment is declared successful, it is important that the old host be
+unregistered with `*subscription-manager clean*` to prevent the environment from
+being out of compliance.
 ====
 
 endif::openshift-enterprise[]
 
+[[manual-upgrades-preparing-for-upgrade]]
 === Preparing for Upgrade
 
-xref:#upgrading-masters[After the master and etcd servers have been upgraded]
-you will need to ensure that your current production nodes are labeled either
-blue or green.  In this example the current installation will be blue and the
-new environment will be green.
+xref:#upgrading-masters[After the master and etcd servers have been upgraded],
+you need to ensure that your current production nodes are labeled either blue or
+green. In this example, the current installation will be blue and the new
+environment will be green.
 
 ----
-oc label --all nodes color=blue
+$ oc label --all nodes color=blue
 ----
 
-There is quite a bit of flexibility here.  Perhaps there was only some subset of
-nodes that required the uptime guarantees of a blue-green deployment.  In that
+There is quite a bit of flexibility here. Perhaps there is only some subset of
+nodes that require the uptime guarantees of a blue-green deployment. In that
 case, the `-l` flag can be used to match a subset of the environment using a
 selector.
 
 Next, xref:../install/advanced_install.adoc#adding-nodes-advanced[the green
-environment] should be created for any nodes that are to be replaced.  The
-`*color=green*` label can be applied by Ansible using the `*openshift_node_labels*`
-variable for each node.  The most critical attribute of the node at this step
-is that the `*openshift_schedulable=false*` variable be set.  The goal is for no
-workloads to be scheduled until these nodes are determined to be healthy.
+environment] should be created for any nodes that are to be replaced. The
+`*color=green*` label can be applied by Ansible using the
+`*openshift_node_labels*` variable for each node. The most critical attribute of
+the node at this step is that the `*openshift_schedulable=false*` variable be
+set. The goal is for no workloads to be scheduled until these nodes are
+determined to be healthy.
 
```

Comment:
* Maybe convert from "The ... can be applied by Ansible..." to "Ansible can apply ..."
* The two sentences following that can be combined, maybe "In order to delay workload scheduling until the nodes are healthy, be sure to set var ..." (with "healthy" being a link to the health checks stuff, perhaps).